readme.md

```diff
@@ -313,6 +361,29 @@ Image attachment ID of the original image or 0.
 
 `wp_calculate_image_sizes()`
 
+##### Usage Example
+
+```
+<?php
+// Constrain the width of full size images to the content width.
+
+function custom_wp_calculate_image_sizes( $sizes, $size, $image_src, $image_meta, $attachment_id ) {
+	if ( is_array( $size ) ) {
+		global $content_width;
+		$width = $size[0]
+
+		if ( ( $width > $content_width ) && ( $width === $image_meta['width'] ) ) {
```

Comment:
Same as above, I would explicitly check the image file name against $image_meta['file'] rather than relying on the width as a size identifier.